inference-engine/thirdparty/clDNN/tests/test_cases/gemm_gpu_test.cpp

```diff
@@ -3536,7 +3536,7 @@ class GemmBaseTest : public ::testing::TestWithParam<gemm_params> {
         EXPECT_EQ(output_ptr.size(), (size_t)(p.b_out_num * p.f_out_num * p.m_size * p.n_size));
         if (sizeof(input0_type) == 1) {
             for (size_t i = 0; i < out_data.size(); ++i) {
-                EXPECT_FLOAT_EQ(float(output_ptr[i]), float(out_data[i])) << "index = " << i;
+                EXPECT_NEAR(float(output_ptr[i]), float(out_data[i]), 1e-1) << "index = " << i;
```

Comment:
why?

Reply:
The values are slightly different and the gemm_gpu_test test fails on the CI.

Reply:
How the region yolo changes affect gemm tests? Is this deviation caused by different random input?

Reply:
Yes, there are random inputs as these.
```
VVVVF<input0_type> input0_data = generate_random_4d<input0_type>(p.b0_num, p.f0_num, x0_size, y0_size, p.range0[0], p.range0[1], p.range0[2]);
VVVVF<input1_type> input1_data = generate_random_4d<input1_type>(p.b1_num, p.f1_num, x1_size, y1_size, p.range1[0], p.range1[1], p.range1[2]);
VVVVF<input2_type> input2_data = generate_random_4d<input2_type>(p.b2_num, p.f2_num, x2_size, y2_size, p.range2[0], p.range2[1], p.range2[2]);
```

The different value was 0.03 when the CI test was failed.